server/__tests__/configuration.test.js

```diff
@@ -290,6 +291,84 @@ describe('Test Meilisearch plugin configurations', () => {
     expect(settings).toEqual({})
   })
 
+  test('Test configuration to remove unpublished entries', async () => {
+    const customStrapi = createFakeStrapi({
+      restaurantConfig: {},
+    })
+
+    const contentType = 'restaurant'
+    const meilisearchService = createMeilisearchService({
+      strapi: customStrapi,
+    })
+
+    const entries = meilisearchService.removeUnpublishedArticles({
+      contentType,
+      entries: [
+        { id: 1, publishedAt: null },
+        { id: 2, publishedAt: null },
+      ],
+    })
+
+    expect(entries).toEqual([])
+  })
+
+  test('Test configuration to unwanted locale entries', async () => {
+    const customStrapi = createFakeStrapi({
+      restaurantConfig: {
+        entriesQuery: {
+          locale: 'fr',
+        },
+      },
+    })
+
+    const contentType = 'restaurant'
+    const meilisearchService = createMeilisearchService({
+      strapi: customStrapi,
+    })
+
+    const entries = meilisearchService.removeLocaleEntries({
+      contentType,
+      entries: [
+        { id: 1, locale: 'fr' },
+        { id: 2, locale: 'en' },
+      ],
+    })
+
+    expect(entries).toEqual([{ id: 1, locale: 'fr' }])
+  })
+
+  test('Test configuration to keep unpublished entries', async () => {
```

Comment:
I had to read the test to understand what it did :) 
"keep" what? Where? 🤔
Maybe you can rename it `Test configuration to keep unpublished entries indexed when...`, or something else of your choice 😉